pkg/provider/aws/resources/s3.go

```diff
@@ -102,14 +103,52 @@ func (bucket *S3Bucket) Configure(params S3BucketConfigureParams) error {
 	return nil
 }
 
-func NewS3Object(bucket *S3Bucket, objectName string, key string, path string) *S3Object {
-	return &S3Object{
-		Name:          objectSanitizer.Apply(fmt.Sprintf("%s-%s", bucket.Name, objectName)),
-		ConstructsRef: bucket.KlothoConstructRef(),
-		Key:           key,
-		FilePath:      path,
-		Bucket:        bucket,
+//
+//func NewS3Object(bucket *S3Bucket, objectName string, key string, path string) *S3Object {
+//	return &S3Object{
+//		Name:          objectSanitizer.Apply(fmt.Sprintf("%s-%s", bucket.Name, objectName)),
+//		ConstructsRef: bucket.KlothoConstructRef(),
+//		Key:           key,
+//		FilePath:      path,
+//		Bucket:        bucket,
+//	}
+//}
```

Comment:
can we delete these or do we want to wait until its fluly implemented?

Reply:
oops, yeah I just forgot to delete